### PR TITLE
'Works' and 'About Me' sections

### PR DESCRIPTION
**Pull request by kwesiObuobi**
- A Figma design is parsed to create the UI
- The mobile version of the 'Works' and 'About me' sections is completed
- Flexbox is used to place elements on the page
- The site is built with a mobile-first approach
- Uses images and backgrounds to enhance the site's look
- CSS is also used to place elements on the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,10 +63,135 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer</span></h1>
...
+                        <h3 class="project-title">Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul class="project-tags">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">css</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">html</li>
+                        </ul>
+                        <a href="#" class="project-action">See Project</a>
+                    </div>
+                </li>
```
Comment (roshan-bajgain) [inline index.html:88]: - [ ] I would suggest you, please use the button styling: Active, Hover, and Pressed (see design).
- [ ] I would suggest you please implement in all the buttons
### Design
his

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kwesiObuobi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -94,6 +94,201 @@ body {
   text-align: center;
 }
 
+/*
+* RECENT WORKS
+*/
+
+.recent-works {
+  margin: 3rem auto 1rem;
+}
```
[the remark this answers is not in the document]
Comment (kwesiObuobi) [reply style.css]: Thank you @roshan-bajgain. I place a `ul` in that section and it has a grid display for the projects. The projects container has a class of `recent-works-group`, and it is actually a grid.

File: index.html [hunk 3]
```diff
@@ -61,10 +63,135 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer</span></h1>
...
+                <h2 class="recent-works-header">My Recent <br> Works</h2>
+                <img src="assets/Indicator.png" alt="ndicator" class="recent-works-indicator">
+            </div>
+            
+            <ul class="recent-works-group">
+                <li class="recent-works-item">
+                    <img src="assets/img-placeholder.png" alt="img placeholder" class="img-placeholder">
+                    <div class="project-description">
+                        <h3 class="project-title">Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul class="project-tags">
```
[the remark this answers is not in the document]
Comment (kwesiObuobi) [reply index.html:80]: Hi @roshan-bajgain. Thank you for flagging all these. I have made the necessary changes please :)

File: index.html [hunk 4]
```diff
@@ -61,10 +63,135 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer</span></h1>
...
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">css</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">html</li>
+                        </ul>
```
[the remark this answers is not in the document]
Comment (kwesiObuobi) [reply index.html:85]: Wow. Can't believe I left all these unchecked. Haha. Thank you @roshan-bajgain

[the remark this answers is not in the document]
Comment (kwesiObuobi) [reply index.html:85]: Please I have made the change as well

File: index.html (hunk 1 above)
Comment (kwesiObuobi) [reply index.html:88]: Thanks for your feedback @roshan-bajgain. I totally missed that. I have updated it now.

**Comment by kwesiObuobi** [comment]
Hi @roshan-bajgain. Please I have made all the necessary changes and pushed them to this very branch. Can you kindly attest to these changes for me? So that I can request for a re-review. Thanks in advance

**Review by PinkMoon25 — APPROVED** [review]
Hi @kwesiObuobi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linter checks :heavy_check_mark: 
Readme :heavy_check_mark: 
github flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kwesiObuobi** [comment]
Thank you, @PinkMoon25. That's nice to hear

**Pull request merged**